src/jazkarta/easyformplugin/savecontent/browser.py

```diff
@@ -16,3 +21,34 @@ def has_saved_content(self):
 
     def saved_content_url(self):
         return self.context.absolute_url() + '/' + STORAGE_ID
+
+
+class CSVDownload(BrowserView):
+
+    def __call__(self):
+        items_dicts = []
+        keys = OrderedDict()
+        for obj in self.context.objectValues():
+            behavior_assignment = EasyformSchemaBehaviorAssignment(obj)
+            obj_dict = {}
+            for field_name in behavior_assignment.schema():
+                obj_dict[field_name] = getattr(obj, field_name)
```

Comment:
I think we probably want to use the field `title` here rather than the internal id.